### PR TITLE
Updating the VIP..again

### PR DESCRIPTION
**Pull request by acahn**

### CODE REVIEW

**Comment by acahn** [comment]
Updating the Vault Integration Program for partners.

File: website/content/docs/partnerships.mdx [hunk 1]
```diff
@@ -16,31 +16,41 @@ This program is intended to be largely a self-service process with links and gui
...
 
-Vault plugins can be built-in and bundled with the Vault binary, or be external that has to be manually mounted. Built-in plugins are developed by HashiCorp, while external plugins can be developed by HashiCorp, technology partners, or the community. There is a curated collection of all plugins, both built-in and external, located on the [Plugin Portal](/docs/plugins/plugin-portal).
+The second type is where a partner develops a custom plugin. Vault has a secure [plugin](/docs/plugins) architecture. Vault’s plugins are completely separate, standalone applications that Vault executes and communicates with over RPC. 
+
+Plugins can be broken into two categories, Auth and Secrets Engines. They can be built-in and bundled with the Vault binary, or be external that has to be manually mounted. Built-in plugins are developed by HashiCorp, while external plugins can be developed by HashiCorp, technology partners, or the community. There is a curated collection of all plugins, both built-in and external, located on the [Plugin Portal](/docs/plugins/plugin-portal).
```
Comment (calvn) [inline website/content/docs/partnerships.mdx:23]:
```suggestion
Plugins can be broken into two categories, Secrets Engines and Auth Methods. They can be built-in and bundled with the Vault binary, or be external that has to be manually registered. Built-in plugins are developed by HashiCorp, while external plugins can be developed by HashiCorp, technology partners, or the community. There is a curated collection of all plugins, both built-in and external, located on the [Plugin Portal](/docs/plugins/plugin-portal).
```

File: website/content/docs/partnerships.mdx [hunk 2]
```diff
@@ -16,31 +16,41 @@ This program is intended to be largely a self-service process with links and gui
...
 
-**Secrets Engines**: Secrets engines are plugin components which store, generate, or encrypt data. Secrets engines are provided with some set of data that perform actions on that data, and then return a result. Some secrets engines store and read data, like encrypted in-memory data structure, and secrets engines connect to other services. Examples of secrets engines include identity modules of Cloud providers like AWS, Azure IAM models, Cloud (LDAP), database or key management. You can find more information about Vault secrets engines [here](/docs/secrets/).
+**Secrets Engines**: Secrets engines are plugin components which store, generate, or encrypt data. Secrets engines are provided with some set of data, that take some action on that data, and then return a result. Some secrets engines store and read data, like encrypted in-memory data structure, other secrets engines connect to other services. Examples of Secrets Engines include identity modules of Cloud providers like AWS, Azure IAM models, Cloud (LDAP), database or certificate management. You can find more information about Vault Secrets Engines [here](/docs/secrets/).
```
Comment (calvn) [inline website/content/docs/partnerships.mdx:47]: Nit: Should be move this up to be right above auth methods, since both are plugin integrations?

Comment (acahn) [reply website/content/docs/partnerships.mdx:47]: The reason it is at the bottom is it flows with the above diagram. The diagram flows: Auth, Runtime, Audit, SEs

**Review by calvn — APPROVED** [review]

**Pull request merged**